frontend/pulse3d/src/utils/labwareCalculations.js

```diff
@@ -100,4 +100,38 @@ export class WellTitle {
   getWellIndexFromRowColumn(rowIndex, columnIndex) {
     return columnIndex * this.numRows + rowIndex;
   }
+  /**
+   *
+   * @param {string} wellName
+   * @returns {int}
+   */
+  wellNameToIndex(wellName) {
+    const wellNameToIndexMap = {
```

Comment:
This isn't what we want, it should basically be doing to opposite of whatever `wellIndexToName` is doing. We don't need to hardcode a mapping like this